shared/xccdf/system/software/integrity.xml

```diff
@@ -310,6 +310,42 @@ Bugzilla #1275532.
 <ref nist="AC-6,AU-9(1),AU-9(3),CM-6(d),CM-6(3)" disa="1494,1496" pcidss="Req-11.5" cis="1.2.6,6.1.3,6.1.4,6.1.5,6.1.6,6.1.7,6.1.8,6.1.9,6.2.3" srg="SRG-OS-000257-GPOS-00098,SRG-OS-000278-GPOS-00108" cjis="5.10.4.1" cui="3.3.8,3.4.1" />
 </Rule>
 
+<Rule id="rpm_verify_ownership" severity="high" prodtype="rhel7">
```

Comment:
Can this XCCDF rule be removed from rhel6/xccdf/system/software/integrity.xml if it's in shared now?

Reply:
no. RHEL6 does not reference shared XCCDF at all. In addition, shared XCCDF needs fixing to handle a lot of different content.